### PR TITLE
Feature/add ingredient to pantry function

### PR DESCRIPTION
**Pull request by DrewBradley**
### Is this a fix or a feature?
I'm not sure at this point what is real and what isn't. This all seems like a dream.

### What is the change/fix?
Yes

### Where should the reviewer start?
All of scripts.js and all of util.js.

### How should this be tested?
run ```npm start``` for both the project and the API. Open the app on a browser at localhost:8080.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -60,6 +60,29 @@ <h1 class='user-name user-name-small small'>User</h1>
           <img tabindex="0" class="card-picture" src="" alt="Food from recipe">
         </section>
       </template>
+      <template class='directions-template'>
```
Comment (elsafluss) [inline src/index.html:63]: templates are sexy

File: src/util.js [hunk 2]
```diff
@@ -28,17 +28,24 @@ export const getCookbook = () => {
...
     method: "POST",
     headers: {
       'Content-Type': 'application/json'
     },
-    body: JSON.stringify(body)
+    body: JSON.stringify(postBody)
   })
     .then(response => response.json())
-    .then(data => data)
+    .then(data => console.log(data))
```
Comment (colefiscus) [inline src/util.js:46]: I think we can change that console.log?

File: src/scripts.js [hunk 3]
```diff
@@ -197,11 +196,51 @@ function displayDirections(event) {
                 ${instruction.instruction}</li>
                 `
                 )
+
               })
         })
     })
 }
 
+function showIngredientsForm() {
```
Comment (elsafluss) [inline src/scripts.js:205]: so glad Cole got these broken out into smaller functions

File: src/util.js [hunk 4]
```diff
@@ -28,17 +28,24 @@ export const getCookbook = () => {
     .then(cookbook => new Cookbook(cookbook))
 }
 
-export const postData = (api, body) => {
-  return fetch(api, {
+export const postData = (userId, ingId, amount) => {
```
Comment (elsafluss) [inline src/util.js:31]: what a bear

**Review by elsafluss — APPROVED** [review]

**Review by colefiscus — APPROVED** [review]
Almost.. there...

**Pull request merged**